core/src/saros/activities/FileActivity.java

```diff
@@ -116,6 +122,15 @@ public String getEncoding() {
     return encoding;
   }
 
+  /**
+   * Returns the file type for the file.
```

Comment:
Why is the file suffix not sufficient ?

Reply:
Even though you already marked this as resolve, just to be clear if anybody else is wondering: this mainly applies to files without a suffix (not always the case) or with an unknown suffix.

Therfore, this could be categorized as a corner case, but I still want to handle it correctly.